xirr.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>=2.2.2'
 
-  spec.add_dependency 'activesupport', '~> 6.1'
+  spec.add_dependency 'activesupport', '>= 6.1'
```

Comment:
Aren't you missing version bump of the gem itself?

Reply:
yea thats true. will bump that up

Reply:
all sorted https://github.com/smartpension/xirr/pull/8/files#diff-aca12f6f9d2130006c1c7e015d0a9b64fc0b1c9ffa3e5a2a3e417c3efde4f416R3